hazelcast/src/hazelcast/client/spi.cpp

```diff
@@ -1089,11 +1090,13 @@ namespace hazelcast {
                     // for back pressure
                     call_id_sequence_->force_next();
                     invoke_on_selection();
-                    return invocation_promise_.get_future().then(boost::launch::sync,
-                                                               [=](boost::future<protocol::ClientMessage> f) {
-                                                                   call_id_sequence_->complete();
-                                                                   return f.get();
-                                                               });
+                    hazelcast::util::hz_thread_pool p(1);
```

Comment:
Is this being used?

Reply:
this was forgotten to be deleted. good catch, deleted it.